src/provider/model/FinEnvironment.ts

```diff
@@ -133,14 +133,16 @@ export class FinEnvironment extends AsyncInit implements Environment {
             this.windowClosed.emit(identity);
         });
 
-        // Register windows that were running before launching the FDC3 service
-        const windowInfo = await fin.System.getAllWindows();
+        Injector.initialized.then(async () => {
```

Comment:
Yes this is intentional.  Awaiting it `Injector.initialized` will never resolve because it itself awaits this init method.  I will add a comment explaining this above this line